subsys/bluetooth/mesh/cdb.c

```diff
@@ -858,7 +858,7 @@ void bt_mesh_cdb_node_del(struct bt_mesh_cdb_node *node, bool store)
 	}
 
 	node->addr = BT_MESH_ADDR_UNASSIGNED;
-	memset(node->dev_key, 0, sizeof(node->dev_key));
+	memset(&node->dev_key, 0, sizeof(node->dev_key));
```

Comment:
removed removed KEY_REF as well as INVALID_KEY_VAL. INVALID_KEY_VAL is zero in PSA standard. Btw there is no guarantee it will not be changed in some revision of specification. It is Ok for now.